integration-tests/multinode/byzantine_test.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+import os
+import time
+from chainrpc import RPC
+from chainbot import SigningKey
+from common import UnixStreamXMLRPCClient, wait_for_validators, wait_for_port, wait_for_blocks, stop_node, wait_for_tx
+
+'''
+three node, 1/3 voting power each.
+target node: node2
+target node mnemonic.
+configs:
+- missed_block_threshold
+- jail_duration
+
+procedure:
+- restore node2 wallet on node0
+- stop node2
+- wait for missed_block_threshold blocks
+- check punishment state on the first staking address
+- start node2
+- wait for node2 port
+- do unjail, expect failure.
+- wait until jail_time + jail_duration
+- do unjail
```

Comment:
this description doesn't seem to match? What is the byzantine fault here?

Reply:
The docstring is copied from jail_test = =".
Updated.